en/docs/setup/configure-analytics-for-elk-stack.md

```diff
@@ -0,0 +1,84 @@
+# Configure Analytics for ELK Stack
+
+## Step 1 - Configure Enforcer
+
+1. Open `apk/helm-charts/templates/data-plane/gateway-components/log-conf.yaml` file.
+
+2. Set following config under `config.toml` section to enable analytics.
+
+    ```toml
+    [analytics]
```

Comment:
shall we add these from values.yaml as well?

Reply:
Added the changes by https://github.com/wso2/docs-apk/pull/203/commits/93d0f12c415a792410bbe134352ab3329fc60847.